utils/custom.js

```diff
@@ -138,8 +152,16 @@ const handleAnonymousData = async (
           is_author: postAuthorId === child.user_id
         };
       }
+
+      let childCommentLv2 = setChildCommentLv2(
+        child?.latest_children?.comment,
+        karmaScores,
+        req.userId,
+        myAnonymousId
+      );
       return {
         ...child,
+        latest_children: {comment: childCommentLv2},
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This section also looks good. The `setChildCommentLv2` function is being used correctly to process level 2 comments for signed-in users. The resulting array is then correctly set as the `latest_children.comment` attribute of the parent comment.


<!-- This is an auto-generated comment by OSS CodeRabbit -->